e2eAIOK/DeNas/pruner/PrunerFactory.py

```diff
@@ -0,0 +1,17 @@
+from e2eAIOK.DeNas.pruner.PytorchPruner import PytorchPruner
+
+PRUNER_BACKENDS = {
+        "pytorch": PytorchPruner,
```

Comment:
add place holder for other plugins, assume INC and other will be plugablehere?

Reply:
added place holder for INC and NNI, will add implementation later.